internal/action/bufpane.go

```diff
@@ -351,16 +349,15 @@ func (h *BufPane) resetMouse() {
 func (h *BufPane) OpenBuffer(b *buffer.Buffer) {
 	h.Buf.Close()
 	h.Buf = b
+	// Set the OverwriteModeActive by default to false on open
```

Comment:
The comment tells *exactly* what is entirely obvious without the comment, and nothing more. (The original comment for `isOverwriteMode` at least told something more; although not exactly a revelation either.)